Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -354,28 +354,7 @@ extension PostCampaignCheckoutViewController: PledgePaymentMethodsViewController
     _: PledgePaymentMethodsViewController,
     didSelectCreditCard paymentSource: PaymentSourceSelected
   ) {
-    switch paymentSource {
```

Comment:
@amy-at-kickstarter reminder for Wednesday - can you reinstate this? Makes more sense than fetching from the graph IMO.